api_end_end_testing/src/main/java/tds/user/api/model/RoleAssociation.java

```diff
@@ -0,0 +1,42 @@
+package tds.user.api.model;
```

Comment:
Normally I would have the code residing in the structure present in this project, but this project is unique as in it will most likely only run via Maven's test lifecycle.  So I would want the Maven directory structure to remain but the `src/main` folders will be empty with everything existing within `src/test/java`.